webapp/packages/plugin-session-expiration/src/SessionExpireWarningDialog/SessionExpireWarningDialogBootstrap.ts

```diff
@@ -66,15 +73,17 @@ export class SessionExpireWarningDialogBootstrap extends Bootstrap {
 
   private async open(): Promise<void> {
     if (!this.dialogInternalPromise) {
-      this.dialogInternalPromise = this.commonDialogService.open(SessionExpireWarningDialog, null);
+      this.dialogInternalPromise = this.commonDialogService.open(SessionExpireWarningDialog, {
+        onButtonClick: () => this.clientActivityService.updateActivity(),
```

Comment:
also since we control activity each user action, there is no need for this due to the fact `touchSession` will be executed because of mouse being moved or clicked

but we can leave it like this just in case if something went wrong in `useClientActivity` hook

Reply:
actually it's must work like this:
each 1m execute `touchSession` if user active
execute `touchSession` if user closed or resolved `SessionExpireWarningDialog` dialog before session expiration